### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [melisgl/mgl-mat#6]
Building with SBCL 2.3.1.122-1455d60f6 / ASDF 3.3.5 for quicklisp dist creation.

Trying to build commit id 2095301b3c5b58cdf3b15a8490034080f8458340

*mgl-mat* fails to build with the following error:

```
; caught ERROR:
;   READ error during COMPILE-FILE: Package CLNU does not exist. Line: 1669, Column: 33, File-Position: 67442 Stream: #<SB-INT:FORM-TRACKING-STREAM for "file /home/quicklisp/quicklisp-controller/dist/build-cache/mgl-mat/a53d2398082fd3643f8eca2ed72e115c2ae856ae/mgl-mat-20230408-git/src/mat.lisp" {1017041363}>
...
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING {1001728003}>: COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "mgl-mat" "src" "mat">

```

*mgl-mat/test* fails to build because of a failure in  _mgl-mat_.

[Full log here](http://report.quicklisp.org/2023-04-08/failure-report/mgl-mat.html)

**Comment by melisgl** [comment]
I can't reproduce it here, but probably some dependency of MGL-MAT used to depend on CL-NUM-UTILS, which I forgot to declare in the ASDF. Hopefully, it's now fixed in melisgl/mgl-mat@835518e.

**Comment by quicklisp** [comment]
Sorry, there's a related problem. cl-num-utils is apparently deprecated and
disappearing and https://github.com/Lisp-Stat/numerical-utilities is the
replacement.

**Comment by melisgl** [comment]
Fixed in ad6fcb5. Tests pass with when using https://github.com/Lisp-Stat/lla instead of the LLA in the current quicklisp dist.
